Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
+
+RUN apt-get update -y
+RUN apt-get install apt-transport-https ca-certificates gnupg curl python3 python3-pip -y
```

Comment:
Can change to "apt-get install apt-transport-https ca-certificates gnupg curl python3 python3-venv -y"

Can apt install python3-venv early on, and don't need to apt install pip it looks like with virtual environments.